events/dlc/ep1/ep1_flavor_events.txt

```diff
@@ -2397,6 +2397,7 @@ scripted_effect ep1_flavor_2000_type_effect = {
 					OR = {
```

Comment:
You missed a few parts in this file.
1. Set the flag, `flag:camel_cavalry`, etc. are set before. Our flags aren't.
![image](https://user-images.githubusercontent.com/46384992/216824337-d204f578-5212-4bdf-8460-61824f62479f.png)
2. A few usings of such flags in this file. Just seach for `flag:camel_cavalry` and do it by analogy.
![image](https://user-images.githubusercontent.com/46384992/216824405-fba33c20-6a5f-460d-be37-2fa9bb81198a.png)
3. `01_ep1_custom_loc.txt`, another file with `flag:camel_cavalry`. The same, do it by analogy.